src/Compilers/CSharp/Portable/FlowAnalysis/NullableWalker.cs

```diff
@@ -43,6 +43,18 @@ internal NullableAnalysisData(int maxRecursionDepth = -1)
             }
         }
 
+        internal readonly struct GetterNullResilienceData(SynthesizedBackingFieldSymbol field, NullableAnnotation assumedAnnotation)
```

Comment:
This is pretty much the same as the tuple we had before but it felt like as the usage was starting to leak into more places, it was better off as a named type.